apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetDAO.java

```diff
@@ -84,4 +86,12 @@ List<Dataset> find(@Bind("limit") int limit,
     int[] recordExperiments(@Bind("workspace_id") String workspaceId,
             @BindMethods Collection<DatasetLastExperimentCreated> datasets);
 
+    @SqlQuery("SELECT workspace_id, created_by AS user, COUNT(DISTINCT id) AS count " +
+            "FROM datasets " +
+            "WHERE DATE(created_at) = DATE(NOW() - INTERVAL 1 DAY) " +
+            "GROUP BY workspace_id,created_by")
+    List<BiInformationResponse.BiInformation> getExperimentBIInformation();
+
+    @SqlUpdate("UPDATE datasets SET created_at = TIMESTAMPADD(DAY, -:days, created_at) WHERE workspace_id=:workspace_id")
```

Comment:
What is the context for this? Is it just for testing? If that is the case, this should be done in the tests via jdbi

Reply:
Yes, just for testing. Refactored.